### PR TITLE
Feature/only exit account

### LINKED ISSUE #41: exit-only rollup account
## Summary
exit-only rollup accounts accumulate all the transactions received in `exitBalance`. In order to detect this new type of account, we set a special babyjubjub, `0x000...0001`, to identify this accouunt type.

## Brief description
- `rollup-tx.circom`: needs to detect only-exit accounts and increase `exitBalance` instead of `balance`
- `rollup-tx-states.circom`: add `isOnlyExit` signal to check account type. This signal would be used afterwards in `balance-updater.circom` to select the proper balance to update

### PR DESCRIPTION
**Pull request by krlosMata**
This PR does the following:

- implements new acocunt type only-exit
  - all transfers to this account type modifies the `exitBalance`

Specification could be found [here](https://github.com/hermeznetwork/docs/blob/feature/update-v1/docs/developers/protocol/hermez-protocol/protocol.md#only-exit-rollup-account)

Closes #41 

## NOTE
this PR should be merged after #47

### CODE REVIEW

File: src/rollup-tx-states.circom [hunk 1]
```diff
@@ -27,6 +29,7 @@ include "../node_modules/circomlib/circuits/poseidon.circom";
  * @output P2_fnc0 - {Bool} - processor 2 bit 0 functionality
  * @output P2_fnc1 - {Bool} - processor 2 bit 1 functionality
  * @output isExit - {Bool} - determines if the transaction is an exit
+ * @output isOnlyExit - {Bool} - determines if the receiver account is an only-exit tyoe account
```
Comment (laisolizq) [inline src/rollup-tx-states.circom]: tyoe --> type

File: js/upgrade-db.js [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+const poseidonHash = require("circomlib").poseidon;
+
+/**
+ * Upgrade rollupDb account to v1
+ */
+class UpgradeDb {
+
+    /**
+     * constructor
+     * @param {Object} rollupDb - RollupDb with ols accounts
```
Comment (laisolizq) [inline js/upgrade-db.js]: ols --> old

**Review by laisolizq — APPROVED** [review]
:smile_cat:

**Review by jesusholoo — APPROVED** [review]
great!

**Pull request merged**